libraries/rush-lib/src/logic/installManager/RushInstallManager.ts

```diff
@@ -438,8 +440,8 @@ export class RushInstallManager extends BaseInstallManager {
    *
    * @override
    */
-  protected canSkipInstall(lastModifiedDate: Date, subspace: Subspace): boolean {
-    if (!super.canSkipInstall(lastModifiedDate, subspace)) {
+  protected canSkipInstall(lastModifiedDate: Date, subspace: Subspace, variant: string | undefined): boolean {
```

Comment:
```suggestion
  protected canSkipInstall(lastModifiedDate: Date, subspace: Subspace, variant?: string): boolean {
```

Reply:
@iclanton  Now that we have subspaces, the variants feature should be subspace-specific.  For example, maybe one subspace is using variants, not another is not.

If we go with that approach, then perhaps the `variant` string could become a property of the `Subspace` class, and then we would avoid the need to pass around this extra parameter.

Reply:
That's a good idea.

Reply:
Unfortunately, subspaces are initialized in `RushConfiguration`, which doesn't know which variant is requested or currently installed during initialization time.